stablehlo/dialect/VhloOps.td

```diff
@@ -324,6 +324,10 @@ def VHLO_ReplicaIdOpV1 : VHLO_Op<"replica_id"> {
   let results = (outs TensorOf<[UI32]>);
 }
 
+def VHLO_PartitionIdOpV1 : VHLO_Op<"partition_id"> {
```

Comment:
Since op was not in the base opset this is batched into the `0.4.0` release:

`VHLO_Op<"partition_id", "0.4.0">`